tests/page-objects/forms/concatenate-strings.wdio.page.js

```diff
@@ -50,20 +48,22 @@ const docs = [
       }
     }
   }];
+const submitButton = () => $('[ng-click="onSubmit()"]');
+const reportContent = () =>$('div#reports-content');
```

Comment:
Please use consistent spacing here: 
```suggestion
const reportContent = () => $('div#reports-content');
```